### ISSUE TITLE
Unable to use CASAuthoritative directive

### ISSUE THREAD

**Issue by mostolog** [Jasig/mod_auth_cas#99]
Hi

As Debian repositories have **mod_auth_cas version 1.0.9** and seems there are some issues with the _Require cas-attribute_ directive that were fixed on 1.1 (https://github.com/Jasig/mod_auth_cas/issues/57) resulting in: _Unknown Authz provider: cas-attribute_, we decided to build and copy the library to our apache 2.4 server.

Just built _mod_auth_cas.so_ on another system and copied to /usr/lib/apache2/modules/mod_auth_cas.so
ldd show's everything is ok.

Although Require cas-attribute now doesn't throw anything, setting _CASAuthoritative On_ on apache configuration complains with:
>Invalid command 'CASAuthoritative', perhaps misspelled or defined by a module not included in the server configuration

Thers's something we can do to help fixing it?

Would it be possible you become the mantainers for Debian/Ubuntu repos?

Regards

**Comment by dhawes** [comment]
CASAuthoritative is not available when using Apache 2.4. We should definitely update the documentation to reflect this.

Apache 2.4 added the RequireAny, RequireNone, and RequireAll directives which allows much greater control over what authorization is used. See the Authorization section of https://httpd.apache.org/docs/current/upgrading.html.

Where you were using CASAuthoritative before, you probably want to use RequireAll now.

**Comment by mostolog** [comment]
Thanks a lot.

I'll let the issue open as a reminder to update documentation, if you don't mind.

BTW: As I have looking for help in several ways without succeeding, I'll dare to ask here:
```
Hi


We're just starting with Apache 2.4 + mod_auth_cas.

So far, we have been able to build it and use Require valid-user and cas-attribute directives.

We'll like to forward cas-attributes to applications using HTTP headers.

Is there any way to achieve this at Apache configuration level/httpd.conf?
Something like: 
> Header echo HTTP_CAS_ATTRIBUTE?

How PHP does retrieve that information as a header? We have tried echoing headers starting with HTTP, CAS...without succeeding.

Are we missing something?


Regards
```
